Library/Formula/pam_u2f.rb

```diff
@@ -0,0 +1,41 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
```

Comment:
We need a stable, tagged release for inclusion in Homebrew. The tag can come from a git checkout, as in:

```ruby
url "https://github.com/fun/fun.git,
    :tag => "1.0.2",
    :revision => "12345569104203920213940203"
```

But we require a stable tagged release.